### PR TITLE
[City] Make fairs field a connection.

### PR DESCRIPTION
**Pull request by alloy**
Closes https://artsyproduct.atlassian.net/browse/LD-19

### CODE REVIEW

**Comment by alloy** [comment]
Note that this is a breaking schema change, so current emission/eigen builds won’t work against staging any longer. I'll be following up with a Emission PR.

File: src/lib/loaders/loaders_without_authentication/gravity.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default opts => {
     fairBoothsLoader: gravityLoader(id => `fair/${id}/shows`, {}, { headers: true }),
     fairPartnersLoader: gravityLoader(id => `fair/${id}/partners`, {}, { headers: true }),
     fairLoader: gravityLoader(id => `fair/${id}`),
-    fairsLoader: gravityLoader("fairs"),
+    fairsLoader: gravityLoader("fairs", {}, { headers: true }),
```
Comment (ashkan18) [inline src/lib/loaders/loaders_without_authentication/gravity.ts]: 👍

File: src/schema/city/index.ts [hunk 2]
```diff
@@ -74,9 +73,9 @@ const CityType = new GraphQLObjectType<any, ResolverContext>({
         }
 
         const response = await showsWithHeadersLoader(gravityOptions)
-        const { headers, body: cities } = response
+        const { headers, body: shows } = response
```
Comment (ashkan18) [inline src/schema/city/index.ts]: 👍

**Comment by ashkan18** [comment]
hehe saw your comment after submitting reviews 👍

File: src/schema/__tests__/city.test.ts [hunk 3]
```diff
@@ -106,7 +106,7 @@ describe("City", () => {
 
       mockShows = [{ id: "first-show" }]
       mockShowsLoader = jest.fn(() =>
-        Promise.resolve({ body: mockShows, headers: { "x-total-count": 1 } })
+        Promise.resolve({ body: mockShows, headers: { "x-total-count": "1" } })
```
Comment (anandaroop) [inline src/schema/__tests__/city.test.ts]: Yep seems proper

**Comment by alloy** [comment]
Works for me 👍

File: src/schema/fairs.ts [hunk 4]
```diff
@@ -60,7 +60,8 @@ const Fairs: GraphQLFieldConfig<void, ResolverContext> = {
       gravityOptions.id = options.ids
       delete gravityOptions.ids
     }
-    return fairsLoader(gravityOptions)
+    const { body: fairs } = await fairsLoader(gravityOptions)
+    return fairs
```
Comment (anandaroop) [inline src/schema/fairs.ts]: Hmm, was this simply returning a Promise before this change 🤔 ?

Comment (alloy) [reply src/schema/fairs.ts]: Correct

**Comment by alloy** [comment]
Alright, the emission PR is up https://github.com/artsy/emission/pull/1359

**Pull request merged**